adoc/TRD-SLES-SAP-HA-automation-quickstart-cloud.adoc

```diff
@@ -30,17 +34,26 @@ image::TRD_SLES-SAP-HA-automation-quickstart-cloud-aws-automation-architecture.j
 endif::[]
 
 ifeval::[ "{cloud}" == "GCP" ]
-GCP
+image::TRD_SLES-SAP-HA-automation-quickstart-cloud-gcp-automation-architecture.png[width=470]
 endif::[]
 
-The project will perform the following:
+The project will perform the following actions:
 
-* Deploy infrastructure - including VPC, subnets, IAM roles etc.
-* Deploy instances -  2 x SAP HANA instances
-* Configure the operating system for the SAP workload
-* Run the SAP HANA installation
-* Configure SAP HANA System Replication (HSR)
-* Configure SUSE High Availability cluster components and resources
+ifeval::[ "{cloud}" == "Azure" ]
+* AZURE
```

Comment:
why you do not add the sentence for azure too?  Now the document shows "Azure" instead of a sentence....